dashboard/config/initializers/fast_localization.rb

```diff
@@ -1,7 +1,10 @@
 # Because loading YAML locales is super-slow, only load english yml locale files in development
 # To load all locales for testing, add "load_locales: true" to locals.yml config
-if Rails.env.development? && (!CDO.load_locales)
-  locale_paths = Dashboard::Application.paths['config/locales'].expanded.select{|x| x.include?('en')}
+
+if (Rails.env.test? or Rails.env.development?) && (!CDO.load_locales)
```

Comment:
if you look at test/ui/i18n.feature for one, it uses spanish and portuguese. I don't remember if there are other UI tests that use other locales (@tanya?). In any case it might be nice to have the ui tests act more production-like -- if only there was a way to easily tell if you were running unit tests or were a server...